tests/trainers/java/time.java

```diff
@@ -0,0 +1,68 @@
+/**
+Test to check the Java Timming. This cover only the adquisition
+of the current date in different ways (because is dependent of
+the system ), but not all the operations can be performed
```

Comment:
I don't understand the docstring. What do you mean by "not all the operations can be performed"? Are they not performed? Why not? And what is the purpose of the list?

Reply:
is it better now? we can remove this comment if is necessary